src/MovieDetail/MovieDetail.js

```diff
@@ -0,0 +1,21 @@
+import './MovieDetail.css';
+
+const MovieDetail = (props) => {
+
+  return (
+    <article className="movieDetail">
+      <div className="title">
+        <h2>{props.details.title}</h2>
+        <p>{props.details.release_date}</p>
+      </div>
+      <div className="details">
+        <p>{`Tagline: ${props.details.tagline}`}</p>
+        <p>{`Runtime: ${props.details.runtime}`}</p>
+        <p>{`Budget: ${props.details.budget}`}</p>
+      </div>
+      <p>{`Overview: ${props.details.overview}`}</p>
```

Comment:
I wonder if there is a way to minimize the amount of times you have to type `props.details`